cmd/endpoint.go

```diff
@@ -133,6 +133,11 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 		}
 	} else {
 		u = &url.URL{Path: path.Clean(arg)}
+		// Check if the arg is a localhost or an IP without a scheme, then ask user to start with a scheme.
+		if strings.Contains(arg, "localhost") || checkIPv4(arg) {
```

Comment:
1. Any reason why `localhost` in entire path (substring) is treated as invalid?
2. It is good to do this check before `u = &url.URL{Path: path.Clean(arg)}`